node/cli/src/command.rs

```diff
@@ -100,11 +200,41 @@ impl SubstrateCli for Cli {
 	}
 
 	fn native_runtime_version(spec: &Box<dyn ChainSpec>) -> &'static RuntimeVersion {
-		match spec.identify() {
-			ChainIdentity::Frequency => &frequency_runtime::VERSION,
-			ChainIdentity::FrequencyRococo => &frequency_rococo_runtime::VERSION,
-			ChainIdentity::FrequencyLocal => &frequency_rococo_runtime::VERSION,
+		if spec.id() == "frequency" {
+			if cfg!(feature = "frequency") {
```

Comment:
IMO This looks too messy, there might be a way to organize these in a cleaner way. Generally speaking most of changes on `command.rs` might have some potential for improvement.

One question might worth asking is that how much we want to optimize the binary. Like there might be an argument to include some of those lines for the sake of readability if added number of lines are not that significant.

Reply:
@aramikm  yes, there will be another PR for this story that will further cleanup